### PR TITLE
[CHKDSK] Implement translation files for Romanian & Italian

### PR DESCRIPTION
**Pull request by Fraizeraust**
Cc @ElCresh for review.

### CODE REVIEW

File: base/system/chkdsk/lang/ro-RO.rc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Adaptarea pentru ReactOS din 1999 de către Emanuele Aliberti\n\n"
+END
+
+STRINGTABLE
+BEGIN
+    IDS_VOLUME_IN_USE "Volumul este folosit și nu poate fi blocat\n"
+    IDS_CHKDSK_FAIL "Operația Chkdsk-ului a eșuat.\n\n"
+    IDS_NO_ENTRY_POINT "Găsirea punctelor de intrare FMIFS este imposibilă.\n\n"
+    IDS_BAD_ARGUMENT "Argument necunoscut: %s\n"
+    IDS_NO_CURRENT_DIR "Obținerea folder-ului respectiv este imposibilă. Cod de eroare: "
```
Comment (reactos-contributor) [inline base/system/chkdsk/lang/ro-RO.rc]: Please translate "directory" as "director". BTW, this term is used in command line context only. Please translate the term "folder", used everywhere else, as "dosar".
I very much insist on this one.

File: base/system/chkdsk/lang/ro-RO.rc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    IDS_FILE_SYSTEM "Tipul sistemului de fișier este %s.\n"
+    IDS_ABOUT "\n\
+Chkdskx v1.0.1 de către Mark Russinovich\n\
+Systems Internals - http://www.sysinternals.com\n\
+Adaptarea pentru ReactOS din 1999 de către Emanuele Aliberti\n\n"
+END
+
+STRINGTABLE
+BEGIN
+    IDS_VOLUME_IN_USE "Volumul este folosit și nu poate fi blocat\n"
```
Comment (reactos-contributor) [inline base/system/chkdsk/lang/ro-RO.rc]: The state of the volume is presented less clear than in its English text. Please consider a few options to improve that aspect: "este folosit la moment", "este în folosință", "este în uz", "este folosit de alt program".

Comment (Fraizeraust) [reply base/system/chkdsk/lang/ro-RO.rc]: I'd rather use `este în uz`. "folosit de alt program" is already used in `IDS_VOLUME_IN_USE_PROC` resource ID.

File: base/system/chkdsk/lang/ro-RO.rc [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+ 
+LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "\n\
+Uz: %s [drive:] [-F] [-V] [-R] [-C] \n\n\
+[drive:]    Specifică modul pilot pentru control.\n\
```
Comment (reactos-contributor) [inline base/system/chkdsk/lang/ro-RO.rc]: I should have looked more carefully at the the English text when I've made my previous comment, and I apologize for that. "Specifies the drive to check" does not have anything to do with a "driver" but a "drive", which is a different concept. First is an intermediary program used to access real or emulated hardware, whereas the second one is a representation for a given information storage unit. Here we may have as well "Specifies the storage unit to check", translatable as "Specifică unitatea de stocare pentru control" (or just "Specifică unitatea de stocare").
I hope you'll still use the term "modul pilot" or "pilot" for "driver" when you'll encounter that term.

File: base/system/chkdsk/lang/ro-RO.rc [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+-F          Rezolvă erorile discului.\n\
+-V          Vizualizează calea tuturor fișierelor de pe disc.\n\
+-R          Găsește sectoarele defecte și recuperează informațiile care pot fi citite.\n\
+-C          Controlează driver-ul doar dacă este necesar.\n\
```
Comment (reactos-contributor) [inline base/system/chkdsk/lang/ro-RO.rc]: I did not comment previously also for this line but it attracts the same kind of objections for its use of "driver" term.
You probably wanted to say "Controlează unitatea de stocare doar dacă este necesar".

File: base/system/chkdsk/lang/ro-RO.rc [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+Chkdskx v1.0.1 de către Mark Russinovich\n\
+Systems Internals - http://www.sysinternals.com\n\
+Adaptarea pentru ReactOS din 1999 de către Emanuele Aliberti\n\n"
+END
+
+STRINGTABLE
+BEGIN
+    IDS_VOLUME_IN_USE "Volumul este în uz și nu poate fi blocat\n"
+    IDS_CHKDSK_FAIL "Operația programului Chkdsk a eșuat.\n\n"
+    IDS_NO_ENTRY_POINT "Nu s-a putut găsi punctele de intrare FMIFS.\n\n"
```
Comment (reactos-contributor) [inline base/system/chkdsk/lang/ro-RO.rc]: Grammar: number disagreement between the verb "a putea" and the noun "punct". (Correct: "Nu s-au putut găsi punctele de intrare FMIFS.")

File: base/system/chkdsk/lang/ro-RO.rc [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    IDS_BAD_ARGUMENT "Argument necunoscut: %s\n"
+    IDS_NO_CURRENT_DIR "Obținerea dosarului respectiv este imposibilă. Cod de eroare: "
```
Comment (reactos-contributor) [inline base/system/chkdsk/lang/ro-RO.rc]: Chkdsk is a command line program, so please use "director" term. Also, please rephrase the message like you did for IDS_NO_ENTRY_POINT and IDS_NO_QUERY_VOL texts.

File: base/system/chkdsk/lang/ro-RO.rc (hunk 5 above)
Comment (Fraizeraust) [reply base/system/chkdsk/lang/ro-RO.rc]: *facepalm*

I need to fix that ASAP. Thank you for pointing it out.

**Comment by ElCresh** [comment]
Sorry but my revirew was saved as a draft and not submitted :)

**Comment by HBelusca** [comment]
Github says that "base/system/chkdsk/chkdsk.rc" conflicts. Can this be fixed?

**Comment by Fraizeraust** [comment]
Conflict fixed. Thank you for noticing it Hermes.

**Pull request merged**